xarray/core/dataset.py

```diff
@@ -2431,6 +2431,25 @@ def apply(self, func, keep_attrs=False, args=(), **kwargs):
         attrs = self.attrs if keep_attrs else None
         return type(self)(variables, attrs=attrs)
 
+        Examples
```

Comment:
This needs to be in the docstring, not the python code!

Reply:
I shouldn't do PR's on a Friday afternoon!